content/docs/csidriver/features/powerflex.md

```diff
@@ -385,6 +385,18 @@ The CSI PowerFlex driver version 1.3 and later support the automatic deployment
   Refer to https://hub.docker.com/r/dellemc/sdc for supported OS versions.
 - There is no automated uninstallation of the SDC kernel module. Follow PowerFlex SDC documentation to manually uninstall the SDC driver from the node. 
 
+From CSM 1.12.0, you can disable automatic SDC deployment.
+
+If you have PowerFlex 4.x with NAS enabled, SDC deployment isn’t needed. By default, SDC deployment is enabled. To disable it, set the `sdc` enabled field to `false`.
```

Comment:
I would remove with "If you have PowerFlex 4.x with NAS enabled, SDC deployment isnt needed". It would be sufficient to say, if you do not want to deploy SDC with PowerFlex driver, it can be disabled by ....